internal/future/serving/tls_util.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 Google LLC
```

Comment:
I think these change can happen in multiple steps.
* Move files to /rpc/ directory. You can drop future since we are deleting the obsoleted code at this point.
* Break out the TLS util stuff
* Add client code.

Reply:
I'll drop future when the original serving package is not being referenced at all. We still have some example codes that are using it.